src/cli/node.ts

```diff
@@ -41,13 +36,6 @@ export const node = async () => {
     );
   }
 
-  // REPL with `ts-node` to support import statements.
-  return tsNode
-    .createRepl({
-      service: tsNode.register({
-        require: ['dotenv/config', 'tsconfig-paths/register'],
```

Comment:
Is the thinking that these don't matter for a REPL, or is the functionality baked in to tsx? (if we're keeping a REPL around, being able to `import 'src/api/blah'` could still be nice...)

Reply:
There is no thinking 😂 I need to test this still. Good question though